src/plonk/prover.rs

```diff
@@ -35,7 +37,7 @@ pub fn create_proof<
     params: &Params<C>,
     pk: &ProvingKey<C>,
     circuits: &[ConcreteCircuit],
-    instances: &[&[Polynomial<C::Scalar, LagrangeCoeff>]],
+    instances: &[&[&[C::Scalar]]],
```

Comment:
Document that the inner columns (the innermost slice) does not need to be a full column; `create_proof` zero-extends all columns (by virtue of copying the values into an initialized `Polynomial`).